### PR TITLE
Advertisement links

### PR DESCRIPTION
**Pull request by wilco375**
### Summary
Add new advertisement links

image

### CODE REVIEW

File: app/styles/components/advertisement-tool.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // CSS3 code for animation of advertisements
-$amount-of-advertisements: 9;
+$amount-of-advertisements: 10;
```
Comment (Matthijsy) [inline app/styles/components/advertisement-tool.scss:2]: You added 3 new advertisements right? So then it should be 12

Comment (Matthijsy) [reply app/styles/components/advertisement-tool.scss:2]: Ah i now see that it is 1 advertisement containing of multiple sub-adverstiements

File: app/templates/components/advertisement-tool.hbs [hunk 2]
```diff
@@ -2,9 +2,17 @@
   <div class="tool adv-container">
     <div class="d-flex adv-row align-items-center">
       {{#each advertisements as |advertisement index|}}
-        <a class="advertisement-item col-auto" href={{advertisement.link}} title={{advertisement.title}} target="_blank">
-          <img src={{advertisement.image}} alt={{if advertisement.alt advertisement.alt advertisement.title}}>
-        </a>
+        {{#if advertisement.links}}
+          <div class="advertisement-item advertisement-item--links col-auto" href={{advertisement.link}} title={{advertisement.title}} target="_blank">
```
Comment (Matthijsy) [inline app/templates/components/advertisement-tool.hbs:6]: I think the href and titlte are not needed here, since those are not defined in the object

Comment (wilco375) [reply app/templates/components/advertisement-tool.hbs]: Oops, you are right. Resolved.

**Review by Matthijsy — APPROVED** [review]

**Pull request merged**